exporters/otlp/internal/retry/retry_test.go

```diff
@@ -23,28 +23,27 @@ import (
 
 	"github.com/cenkalti/backoff/v4"
 	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 func TestWait(t *testing.T) {
 	tests := []struct {
-		ctx      context.Context
-		delay    time.Duration
-		expected error
+		ctx          context.Context
+		delay        time.Duration
+		expectedToBe error
```

Comment:
nit: This rename doesn't seem needed.

Reply:
Yes, I was thinking that the condition had changed from expected equality to expected "errors.Is" (meaning "to be").